staging/Archivist-Workstation-Processing.md

```diff
@@ -38,13 +38,17 @@ the components, and enter them into ArchivesSpace.
 * Double-click the drive icon to display the drive folders in Windows Explorer.
 * The folders will have the following structure:  
     * \M12345_workingfiles
-		* \M12345-0001
+		* \ACQ_12345_54321
 			* \metadata
-			* \objects 
+			* \objects  
+          	* \data (optional)    
```

Comment:
When do we use a `\data` folder? Should this be a child of `\objects` for bagged transfers?

Reply:
Most file transfers, like optical media, have a data folder. The data folder is displaying as a child of objects in my local version. Unsure why that indent doesn't appear for you.